modules/runner/src/integrationTest/java/org/apache/ignite/internal/AbstractClusterIntegrationTest.java

```diff
@@ -71,7 +75,8 @@ public abstract class AbstractClusterIntegrationTest extends BaseIgniteAbstractT
             + "        gossipInterval: 10\n"
             + "      },\n"
             + "    }\n"
-            + "  }\n"
+            + "  },"
+            + "  cluster.failoverTimeout: 0\n"
```

Comment:
Looks out of place, to be honest. Maybe it should be a part of network configuration, let's consider that a possibility

Reply:
Well, it seems to be about CMG, not about the network. You already suggested that we revisit all configuration properties, we could handle this property there as well.